lib/logstash/util/defaults.rb

```diff
@@ -0,0 +1,16 @@
+# encoding: utf-8
+require "logstash/namespace"
+require "logstash/util"
+
+module LogStash module Util module Defaults
```

Comment:
I think this module should be in `lib/logstash/config/defaults.rb` since its more about default config value than util, what do you think?